src/main/java/org/jabref/gui/preferences/EntryEditorTab.fxml

```diff
@@ -60,8 +58,4 @@
             <Insets left="20.0"/>
         </padding>
     </VBox>
-    <Label styleClass="sectionHeader" text="%Default drag &amp; drop action"/>
-    <RadioButton fx:id="dndCopyFile" text="%Copy file to default file folder" toggleGroup="$dnd"/>
-    <RadioButton fx:id="dndLinkFile" text="%Link file (without copying)" toggleGroup="$dnd"/>
-    <RadioButton fx:id="dndCopyRenameLinkFile" text="%Copy, rename and link file" toggleGroup="$dnd"/>
```

Comment:
Why is the string still in JabRef_en.properties? Shouldn't it disappear there, too?

Reply:
Fixed.